src/app/sentry-example-page/page.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
```

Comment:
is this going to be removed after testing?

Reply:
yes